src/handlers/clean/handler.ts

```diff
@@ -57,8 +56,6 @@ async function cleanRecord(record: SQSRecord): Promise<void> {
     vendor_id: vendorId,
   };
 
-  console.log("Cleaned Body Object:", cleanedBodyObject);
```

Comment:
Can we still log just the vendor_id here? That would still be 'safe' right?

Reply:
I've gone with event_id as that's what we're logging elsewhere